Bio/PDB/Atom.py

```diff
@@ -11,19 +11,15 @@
 import sys
 import warnings
 from typing import Optional
-from typing import TYPE_CHECKING
 from typing import TypeVar
 
 import numpy as np
 
-import Bio.PDB.Atom
```

Comment:
Weird - this was importing itself?

Reply:
That's fascinating. I'm surprised it's not throwing a circular reference error, but I guess maybe it recognizes itself in the namespace and ignores the import? I think that TypeVar below is being created just to try to avoid circular references (I think that was more common when type checkers were less clever), so maybe someone accidentally broke it with a circular reference, and then the person who fixed it didn't clean up all the changes

Reply:
Could be - I didn't examine the history here.

Reply:
Looking the `git blame Bio/PDB/Atom.py`, it looks like this line was added in #4593. My speculation is that the type annotations originally used `Atom` rather than the string `"Atom"` so it was imported, then when the `mypy` complained about using a type within itself the type annotation was changed to a string but removing the import was skipped over. I don't have any evidence for this though :)